tests/rptest/services/flink.py

```diff
@@ -18,6 +18,18 @@
 
 from rptest.services.provider_clients import get_metadata_for_node
 
+# default specs are a fail safe proof variation of minimal VM specs possible
+instance_specs = {
+    "default": {
+        "vcpus": 2,
+        "ram": 16,
+    },
+    "i3en.xlarge": {
```

Comment:
scale tests run on  `is4gen.xlarge` and we use `dedicated_nodes` flag to determine the environment we are in.

Reply:
Will update